ui/selectable.js

```diff
@@ -73,19 +74,18 @@ return $.widget("ui.selectable", $.ui.mouse, {
 		};
 		this.refresh();
 
-		this.selectees = selectees.addClass("ui-selectee");
+		this._addClass( selectees, "ui-selectee" );
+		this.selectees = selectees;
```

Comment:
Could also drop the `selectees` variable and just assign `that.selectees` inside `refresh()` above.